apps/explorer/src/components/HomeMetrics/CurrentEpoch.tsx

```diff
@@ -75,7 +75,10 @@ export function CurrentEpoch() {
                         <Heading variant="heading6/medium" color="steel-darker">
                             {label ?? '--'}
                         </Heading>
-                        <ProgressBar progress={progress || 0} />
+                        <ProgressBar
+                            animate={!!progress && progress > 15}
```

Comment:
do you think it would be reasonable to make the progress bar animated by default if the progress is > 15%? it would also be nice to pull this into a constant :)

Reply:
Yea, I think I will force a constant here. Below 15% it just looks really weird to have the glowing dot basically on 100% of the time